packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -640,6 +656,14 @@ function isCouponApplied( { coupon_savings_integer = 0 }: ResponseCartProduct )
 	return coupon_savings_integer > 0;
 }
 
+function hasPartnerCoupon( coupon: string, products?: ResponseCartProduct[] ): boolean {
+	const productHasSublabel =
+		products && products.find( ( product: ResponseCartProduct ) => getSublabel( product ) );
```

Comment:
Minor: this could use `some` instead of `find`.

```suggestion
		products && products.some( ( product: ResponseCartProduct ) => !! getSublabel( product ) );
```